<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreviewMainWindow</class>
 <widget class="QMainWindow" name="PreviewMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="screenshotButton">
   </widget>
   <widget class="QPushButton" name="refreshButton">
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionSelectSizeMode"/>
  <action name="actionSetSizeModeOff">
   <property name="text">
    <string>Manual zoom</string>
   </property>
   <property name="toolTip">
    <string>Manual zoom</string>
   </property>
  </action>
  <action name="actionSetSizeModeVerticalFit">
   <property name="text">
    <string>Vertical fit</string>
   </property>
   <property name="toolTip">
    <string>Vertical fit</string>
   </property>
  </action>
  <action name="actionSetSizeModeHorizontalFit">
   <property name="text">
    <string>Horizontal fit</string>
   </property>
   <property name="toolTip">
    <string>Horizontal fit</string>
   </property>
  </action>
  <action name="actionSetSizeModeBestFit">
   <property name="text">
    <string>Best fit</string>
   </property>
   <property name="toolTip">
    <string>Best fit</string>
   </property>
  </action>
  <action name="actionSetSizeModeOriginalFit">
   <property name="text">
    <string>Original fit</string>
   </property>
   <property name="toolTip">
    <string>Original fit</string>
   </property>
  </action>
   <action name="actionSelectMSAA"/>
   <action name="actionSetMSAAx0">
     <property name="text">
       <string>MSAA Off</string>
     </property>
     <property name="toolTip">
       <string>MSAA Off</string>
     </property>
   </action>
  <action name="actionSetMSAAx2">
   <property name="text">
    <string>MSAA x2</string>
   </property>
   <property name="toolTip">
    <string>MSAA x2</string>
   </property>
  </action>
  <action name="actionSetMSAAx4">
   <property name="text">
    <string>MSAA x4</string>
   </property>
   <property name="toolTip">
    <string>MSAA x4</string>
   </property>
  </action>
  <action name="actionSetMSAAx8">
   <property name="text">
    <string>MSAA x8</string>
   </property>
   <property name="toolTip">
    <string>MSAA x8</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
